### ISSUE TITLE
Local Directory is empty with FTI_Checkpoint

### ISSUE THREAD

**Issue by Xinying-Wang-CS** [leobago/fti#394]

**Comment by Xinying-Wang-CS** [comment]
Hi, thank you for the excellent work. Just confused about FTI_Snapshot and FTI_Checkpoint. For my understanding, we can set checkpoint frequency with FTI_Snapshot but trigger special checkpoint conditions with FTI_Checkpoint. The thing is, I was struggle with Checkpoint level in FTI_Checkpoint. I'm not sure the difference when I pass  1 and 4 to FTI_Checkpoint. For example, when I set level 1 in FTI_Checkpoint and keep_last_ckpt=1, when it checkpoints, my local node directories always empty (but got L4 fits in global ), I'm confused how can I keep the fit infos in local. Any help from you will be appreciated.

**Comment by leobago** [comment]
Hi Xinying-Wang, the L1 checkpoints are kept into your local storage until the next checkpoint or until the end of the execution. When the execution reaches the end, FTI copies the L1 checkpoints into the PFS, which kind of transforms it into a L4 checkpoint. However, that only happens at the very end of the execution, during the run you should be able to see the local directory with the checkpoint files. I hope this answers your question. Let me know if you have more doubts.

**Comment by Xinying-Wang-CS** [comment]
Thank you for the quick response. Yes, when I set L1 in FTI_Checkpoint and
trigger checkpoint condition, I can only see the ckp infos (L4) in global
and meta, the node directories are empty. If so, how does L1 work? And when
I restart the app, I can see the content grows in these node directories,
but as you said, nothing left after the execution. So my questions are:
1) If I set L1 in FTI_Checkpoint, what does the application depend on for
recovery as I only have L4 infos in global and meta.
2) What is the exact difference between L1 and L4 in FTI_Checkpoint as the
contents in local are empty in the end.

Thank you sooo much.
